tests/integration/test_newrelic.py

```diff
@@ -0,0 +1,50 @@
+from tests.integration import basetest
+
+
+class TestCaseDeployWithNewRelic(basetest.BaseTest):
+    def _deploy_app(self, mda_file, newrelic=True):
+        super().setUp()
+
+        env_vars = {}
+
+        if newrelic:
+            env_vars["NEW_RELIC_LICENSE_KEY"] = "dummy_token"
+            env_vars["NEW_RELIC_METRICS_API"] = "metrics_api"
+            env_vars["NEW_RELIC_LOGS_API"] = "logs_api"
+
+        self.stage_container(mda_file, env_vars=env_vars)
+        self.start_container()
+
+    def _test_fluentbit_running(self, mda_file):
+        self._deploy_app(mda_file)
+        self.assert_app_running()
+
+        # Check if FluentBit is running
+        output = self.run_on_container("ps -ef | grep fluentbit")
+        assert output is not None
+        assert str(output).find("fluent-bit") >= 0
+
+    def _test_fluentbit_not_running(self, mda_file):
+        self._deploy_app(mda_file, newrelic=False)
+        self.assert_app_running()
+
+        # Check if FluentBit is not running
+        output = self.run_on_container("ps -ef | grep fluentbit")
+        assert str(output).find("fluent-bit") == -1
+
+    def _test_newrelic_is_configured(self):
+        self.assert_string_in_recent_logs(
+            "New Relic has been configured successfully."
+        )
+
+    def _test_newrelic_is_not_configured(self):
+        self.assert_string_in_recent_logs("Failed to configure New Relic.")
+
+    def test_newrelic_mx9(self):
+        self._test_fluentbit_running("BuildpackTestApp-mx9-7.mda")
+        self._test_newrelic_is_configured()
+
+    def test_newrelic_not_configured(self):
```

Comment:
There's no app for testing mx10 at the moment. We would have to add that or get someone else to add it.
I think it could be done, but not the scope of our changes now.

Reply:
correct, we'll update all integration tests with mx10 in a separate ticket.